assets/translations/en-us.yaml

```
@@ -782,6 +784,12 @@ monitoring:
       retentionSize: Retention Size
       resourceLimits: Resource Limits
       scrape: Scrape Interval
+      ignoreNamespaceSelectors:
+        label: Use Namespace Selectors
+        radio:
+          enforced: "Use: Monitors can access resources in the namespace specified in the namespaceSelector field"
+          ignored: "Ignore: Monitors can only access resources in the deployed namespace that they are deployed in"
```

```suggestion
          ignored: "Ignore: ServiceMonitors & PodMonitors can only monitor resources in the namespaces they are deployed within. The `namespaceSelector` field will be ignored."
```
Not sure if the \` works as expected, but would be nice if we can highlight `namespaceSelector` to make it clear that it is a field of the ServiceMonitor / PodMonitor spec